org-jira.el

```diff
@@ -362,17 +362,19 @@ See `org-default-priority' for more info."
   "Just do some work on ISSUE-ID, execute BODY."
   (declare (debug t)
            (indent 1))
-  `(let* ((proj-key (replace-regexp-in-string "-.*" "" issue-id))
```

Comment:
We could've also unquoted the `issue-id` here but then this will have the problem that the form would be substituted 3 times... very bad if it has side-effects.